dev/build.image/build.gradle

```diff
@@ -89,18 +89,31 @@ task copySwidTagToBuildImage (type:Copy) {
     dependsOn jar
     from project.file('publish')
     into project.file('wlp/lib/versions/tags')
-    include 'ibm.com_Open_Liberty*.swidtag'
+    include 'ibm.com_Open_Liberty-*.swidtag'
     filter(ReplaceTokens,
-           tokens: [ VERSION: bnd.libertyRelease ])
+           tokens: [VERSION: bnd.libertyRelease ])
```

Comment:
nit - remove this change so that this commit doesn't over write line 94's history.